packages/cli/src/cmds/openapi.ts

```diff
@@ -58,7 +51,6 @@ class OpenAPICommand {
   }
 
   async collectAppMap(file: string): Promise<void> {
-    this.count += 1;
```

Comment:
Is this intentional?  Was this a bug? What was the bug?

Reply:
As far as I can tell it's not used anywhere, I'm not sure what the purpose is.